index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/5116129a4d.js" crossorigin="anonymous"></script>
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css"/>
+</head>
+<body>
+   <section class="header">
+    <div class="name">Daniel</div>
+    <div class="hamburger"><i class="fas fa-hamburger"></i></div>
+   </section>
+
+   <section class="headline">
+    <h1 class="intro">Hey There. I'm Daniel</h1>
+   <p class="developer">I'm a Software developer</p>
+   <p class="abilities">I can help you build a product , feature or website Look through some of my work and experience! 
+    If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+    <div class="icons">
+        <a href="" class="github"><i class="fa-brands fa-github"></i></a>
+        <a href="" class="linkedin"><i class="fa fa-linkedin" aria-hidden="true"></i></a>
+        <a href=""><i class="fab fa-angellist"></i></a>
+        <a href="" class="twitter"><i class="fa-brands fa-twitter"></i></a>
+        <a href="" class="facebook"><i class="fa-brands fa-facebook-f"></i></a>
+    </div>
+   <div class="design"> <img src="images/Illustration.png" alt=""></div>
```

Comment:
- [ ] Kindly use a `background-image` css identifier to create the background image instead of using an `img` tag to add it. If an image is not part of the content (aesthetic reasons) kindly always use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.